QuantConnect.DataBento/DataBentoDataProvider.cs

```diff
@@ -294,8 +232,10 @@ public void Dispose()
         /// <returns>An enumerable of BaseData points</returns>
         public IEnumerable<BaseData>? GetHistory(Data.HistoryRequest request)
         {
+            Log.Trace($"DataBentoProvider.GetHistory(): Received history request for {request.Symbol}, Resolution={request.Resolution}, TickType={request.TickType}");
             if (!CanSubscribe(request.Symbol))
             {
+                Log.Error($"DataBentoProvider.GetHistory(): Cannot provide history for {request.Symbol} with Resolution={request.Resolution}, TickType={request.TickType}");
```

Comment:
You should show it once to avoid spamming.
Please, investigate the Polygon code.: [link](https://github.com/QuantConnect/Lean.DataSource.Polygon/blob/master/QuantConnect.Polygon/PolygonDataProvider.cs#L480)